lib/ruby_lsp/requests/hover.rb

```diff
@@ -21,9 +21,10 @@ class Hover < Listener
 
       ALLOWED_TARGETS = T.let(
         [
-          YARP::ClassNode,
-          YARP::ModuleNode,
+          YARP::CallNode,
```

Comment:
I had used `ClassNode` and `ModuleNode` when rebasing against `main`, which was incorrect – it's the _reference_ to the class or module that we want the hover for.